colorcoder/GenerateManual.java

```diff
@@ -0,0 +1,17 @@
+package colorcoder;
+
+public class GenerateManual {
+
+    public GenerateManual(){
+        
+        System.out.println("These are the color codes\n");
+        System.out.println("PairID ---> MajorColor, MinorColor\n");
+
+        ColorService service = new ColorService();
+        for(int pairnumber = 1; pairnumber <= 25; pairnumber++)
+	    {
```

Comment:
Don't hardcode loop limits. Easy to forget updating this when colors are added or removed.

Reply:
Implemented a new method to first calculate parNumber and then use an existing method to get pair colors using this number.